### PR TITLE
Make it easier to manage Script labels

### PR DESCRIPTION
**Pull request by niklasl**
- Add SPARQL construct for getting them from wikidata
- Add SPARQL utility script for local use
- Update wikidata concept URI:s (their canonical forms use `http`)
- Add simple invocation example to top of file (for maintenance)

### CODE REVIEW

File: source/i18n/scripts.ttl [hunk 1]
```diff
@@ -1,104 +1,108 @@
-# Start here: <https://en.wikipedia.org/wiki/ISO_15924>
+# For a list of available alphabets/scripts, start here:
+#   <https://en.wikipedia.org/wiki/ISO_15924>
+# When you add a new script or need to update all labels; get labels for all by calling:
+#   $ python3 scripts/construct.py source/remote/construct-matching-wd-labels.rq THIS_FILE
+# then add the resulting :prefLabel data here.
```
Comment (kwahlin) [inline source/i18n/scripts.ttl:5]: Maybe add a reminder that there should always be a `@sv`-tagged label

**Review by kwahlin — APPROVED** [review]
LGTM! (At least as a short-term solution)

**Comment by olovy** [comment]
Nice!
I think we should pick the Swedish prefLabels from scripts.ttl to keep the under our control.

File: source/i18n/scripts.ttl (hunk 1 above)
Comment (niklasl) [reply source/i18n/scripts.ttl:5]: Good point; added in 64c600f6316a95f0dbad9422dd8bb7eb64d160e6

**Pull request merged**